deploy/resetEnvironment.js

```diff
@@ -0,0 +1,65 @@
+const process = require("process");
+
+if (process.env["DeployEnvironment"] && process.env["DeployEnvironment"].toLowerCase() === "prod") {
```

Comment:
Why are there 2 conditions for checking the environment? o.O

Reply:
The first makes sure it isn't null, otherwise the second statements invocation of .toLowerCase() would throw.

Reply:
Just avoiding the null reference exception from calling .toLowerCase()